pkg/minikube/constants/constants.go

```diff
@@ -65,9 +68,29 @@ const (
 	DefaultVMDriver = "virtualbox"
 )
 
+var DefaultKubernetesVersion = version.Get().GitVersion
```

Comment:
This will require updating the Makefile to include K8S_VERSION_LDFLAGS when building minikube.